src/style.css

```diff
@@ -0,0 +1,128 @@
+body {
+  /* background-color: red; */
```

Comment:
We can make our codes more nice by removing this comment here

Reply:
Ok thanks for the feedback